src/main/java/frc/robot/jetson/JetsonConnection.java

```diff
@@ -179,33 +197,28 @@ private boolean strToBool(String str) {
          * hub is currently in the turntable blind spot.
          * @return Whether turret vision is working.
          */
-        public boolean turretVisionWorking() { 
-            return turretVisionStatus; 
+        public boolean turretVisionWorking() {
+            return isConnected && turretVisionStatus; 
         }
 
-        /**
-         * Gets the calculated turret angle.
-         * @return The desired turntable angle.
-         */
-        public double getTurretTheta() { 
+        public double getTurretTheta() {
             return turretTheta; 
         }
 
-        /**
-         * Gets the calculated hub distance.
-         * @return The distance from the camera to the hub.
-         */
-        public double getHubDistance() { 
+        public double getHubDistance() {
             return hubDistance; 
         }
```

Comment:
it'd probably be clean to return a pair of (distance, theta) and automatically include setting consumed to true as part of that method